src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -378,7 +378,8 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
                   block.data.hash,
                   block.data.header.level,
                   Some(block.data.header.timestamp.toInstant),
-                  DatabaseConversions.extractCycle(block)
+                  DatabaseConversions.extractCycle(block),
+                  DatabaseConversions.extractPeriod(block.data.metadata)
```

Comment:
I'm not too fond of using what should be an internal utility for DatabaseConversions as a public method.
I'd rather extract the methods to somewhere more self-contained (TezosOptics.Blocks?) or I would create a separate method distinct from that of DatabaseConversions (which should be marked `private`)

Reply:
ok, moved to TezosOptics